discovery-infra/assisted_service_api.py

```diff
@@ -14,9 +18,52 @@ def __init__(self, inventory_url):
         self.inventory_url = inventory_url
         configs = Configuration()
         configs.host = self.inventory_url + "/api/assisted-install/v1"
+        self.set_config_auth(configs)
+
         self.api = ApiClient(configuration=configs)
         self.client = api.InstallerApi(api_client=self.api)
 
+    def set_config_auth(self, c):
+        offline_token = os.environ.get('OFFLINE_TOKEN', "")
```

Comment:
till now all params where passed through command line args but it is ok i think. Just u need to verify what value with u will have here. U don't need to set "" as default value

Reply:
Same thing here with skipper, it's setting the value to `""` if the user leaves it unset so I'm trying to collapse the actually unset case and "skipper unset" case into one condition by setting the default to `""`

Reply:
ok

Reply:
This one resolved @tsorya ?